### ISSUE TITLE
Broken Version 2.8.0: Uncaught TypeError

### ISSUE THREAD

**Issue by thebugfixnet** [WordPress/gutenberg#6583]
## Issue Overview

Version 2.8.0 open an released post with blank editor. Only the title of the article is available. Openning same post with old editor everything works fine.

## Steps to Reproduce (for bugs)

1.  Simple edit an existing post.

Gutenberg V 2.8.0. 
Wordpress 5.0-alpha-43162

Setting: German language

Error message:

[...]/plugins/gutenberg/core-blocks/build/index.js

index.js:12 Uncaught TypeError: Object(...) is not a function
    at Object.<anonymous> (index.js:12)
    at n (index.js:1)
    at index.js:1
    at index.js:1
(anonymous) @ index.js:12
n @ index.js:1
(anonymous) @ index.js:1
(anonymous) @ index.js:1

**Comment by pento** [comment]
Thank you for the bug report, @thebugfixnet! Could I get you to try clearing your browser cache, and then opening the editor again? We've had similar reports where old versions of the JS files are being loaded just after upgrading.

Also, could you try disabling any plugins you have installed, and switching to one of the Twenty* themes briefly, just to see if that makes a difference?

**Comment by thebugfixnet** [comment]
Hi @pento 

I've already cleared the browser cache automatically and manually before I send the report. 

I tried 2.8.0. on three wordpress installations (two actual beta, one on 4.9.5) on the two betas I had the problem. One of the beta installations is a clean Twenty theme test environment. The WP 4.9.5 + Gutenberg 2.8.0. worked fine. 

I tried several typical workarrounds to solve it. But neither clearing the cache nor de-activate / re-activate of the Gutenberg plugin solved it. Neither cookie and deleting all website data in chrome, edge, firefox and opera worked. I also used anonymous browers sessions for an additional test. Also manual refresh of the Gutenberg files by manual ftp upload did not solve the issue.

But finally I get it solved by completely uninstall and delete the stayed data in installation folder of the Gutenberg plugin via ftp and reinstalling it. Btw: It was strange for me that after un-install the folders "core-block", "core-data" and "data" where not deleted.
